sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -455,6 +455,10 @@
               {
                 "value": "studentdisabilityappeal",
                 "label": "Disability Information"
+              },
+              {
+                "value": "studentdependantsappealparttime",
+                "label": "Add/Remove Dependants (Part-Time Applications)"
```

Comment:
As of now the change request forms are going to be generic (at least until MVP as per business) not based upon the offering intensity.

i.e the name or title of the change request forms need to be generic e.g. `Add/Remove Dependents`